pkg/cluster/install.go

```diff
@@ -175,6 +175,7 @@ func (m *manager) Update(ctx context.Context) error {
 		steps.Action(m.configureIngressCertificate),
 		steps.Action(m.updateOpenShiftSecret),
 		steps.Action(m.updateAROSecret),
+		steps.Action(m.renewMDSDCertificate),
```

Comment:
Idea - I think we might want to consider doing `renewMDSDCertificate` earlier in the update code path so that SRE has working logging in the event that the subsequent steps fail.

Maybe we can execute `renewMDSDCertificate` immediately after the `apiServersReady`?

Reply:
@cadenmarchese nice idea, I have made the suggested change.